libs/dyn/yamlsaver/saver_test.go

```diff
@@ -9,40 +9,42 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+var s *saver = NewSaver()
+
 func TestMarshalNilValue(t *testing.T) {
 	var nilValue = dyn.NilValue
-	v, err := ToYamlNode(nilValue)
+	v, err := s.toYamlNode(nilValue)
```

Comment:
Better to initialize for each test in case the saver starts to carry state that can leak between tests.